storage/mysql/log_storage_test.go

```diff
@@ -17,8 +17,6 @@ import (
 	"github.com/google/trillian/testonly"
 )
 
-// TODO(al): add checking to all the Commit() calls in here.
```

Comment:
Yeah it's another of the things I disagree with. If they weren't also against having an assertion based test framework that could be used to write readable tests without haggling over error messages they might have a better case.